src/resolvers.ts

```diff
@@ -22,17 +22,34 @@ export const resolveConfigPath = async (path: Arrayable<string>) => (
  * @param srcDir
  * @returns array of resolved content globs
  */
-export const resolveContentPaths = (srcDir: string) => ([
-  `${srcDir}/components/**/*.{vue,js,ts}`,
-  `${srcDir}/layouts/**/*.vue`,
-  `${srcDir}/pages/**/*.vue`,
-  `${srcDir}/composables/**/*.{js,ts}`,
-  `${srcDir}/plugins/**/*.{js,ts}`,
-  `${srcDir}/utils/**/*.{js,ts}`,
-  `${srcDir}/{App,app}.{js,ts,vue}`,
-  `${srcDir}/{Error,error}.{js,ts,vue}`,
-  `${srcDir}/app.config.{js,ts}`
-])
+export const resolveContentPaths = (srcDir: string, nuxt = useNuxt()) => {
+  const r = (p: string) => p.startsWith(srcDir) ? p : `${srcDir}/${p}`
+  const extensionFormat = (s: string[]) => s.length > 1 ? `.{${s.join(',')}}` : `.${s.join('') || 'vue'}`
+
+  const defaultExtensions = extensionFormat(['js', 'ts', 'mjs'])
```

Comment:
How about jsx/tsx for more niche uses?

Reply:
`nuxt.options.extensions` includes jsx and tsx by default. If a project configures that to be empty (which is unlikely), that's when we use these extensions. 🙂

Reply:
`defaultExtensions` appears to be used directly for certain paths like those in `nuxt.options.imports.dirs` where JSX usage might end up useful, only directories/files that would typically contain SFCs seem to use `nuxt.options.extensions`.

Reply:
I see your point, but that's only there because Nuxt scans for those extensions in `nuxt.options.imports.dirs` - are you using JSX/TSX in those directories and they're scanning? I just tested on Stackblitz; if there's additional configuration you're going, let me know.

> where JSX usage might end up useful

In any case, I can't comment on some usage being useful, because everything can be added in - but this is in the default configuration; the module of course allows you to extend this, so if it is useful for your project, by all means you could add it in the configuration.

Reply:
> that's only there because Nuxt scans for those extensions in `nuxt.options.imports.dirs`

Fair enough! I suppose JSX usage is rare within Nuxt projects anyway